master/pkg/actor/actors/nothing.go

```diff
@@ -0,0 +1,11 @@
+package actors
+
+import "github.com/determined-ai/determined/master/pkg/actor"
+
+// Nothing is a nothing placeholder actor, mainly for when actors are used as IDs and we haven't
+// migrated away from them, yet.
+var Nothing = nothing{}
+
+type nothing struct{}
+
+func (*nothing) Receive(ctx *actor.Context) error { return nil }
```

Comment:
nit: is this actor ever started? just curious if it might be better to panic or return an error here in case we receive something unexpected